package.json

```diff
@@ -1,94 +1,68 @@
 {
   "name": "react-web-booster-kit",
-  "version": "1.0.1",
+  "version": "2.0.0",
   "homepage": "boosters.joinspartan.com",
   "description": "Get started with React, Redux, and React-Router!",
   "main": "index.js",
   "private": "true",
   "engines": {
-    "node": ">=7.4.0"
+    "node": ">=8.3.1"
   },
   "scripts": {
-    "analyze": "flow",
-    "build": "bnr build",
+    "build": "yarn run clean && yarn run init && bnr build",
     "clean:project": "yarn cache clean && yarn install --force",
-    "clean": "rimraf dist",
-    "compile": "bnr compile",
-    "debug:dev": "bnr debug:dev",
-    "debug:prod": "bnr debug:prod",
+    "clean": "rimraf build",
     "deploy:prod": "bnr build && bnr publish:prod",
     "deploy:test": "bnr build && bnr publish:test",
-    "dev": "bnr dev",
-    "flow:update": "flow-typed install --overwrite",
-    "lint:fix": "yarn run lint -- --fix",
-    "lint": "eslint bin build config server universal tests",
-    "preflight": "yarn run analyze && yarn run lint && yarn run test",
+    "init": "node scripts/init",
+    "lint": "tslint -c tslint.json 'src/**/*.{ts,tsx}'",
+    "preflight": "yarn run lint && yarn run test",
     "start": "bnr start",
     "storybook:build": "build-storybook -c .storybook -0 .storybook-static",
     "storybook:start": "start-storybook -p 4003 -c .storybook",
-    "test:watch": "bnr test --watch --config tests/config/config.json",
-    "test": "bnr test --config tests/config/config.json"
+    "test:watch": "bnr test --watch",
+    "test": "bnr test",
+    "test:features": "bnr test:features",
+    "test:specs": "bnr test:specs"
   },
   "betterScripts": {
-    "compile": {
-      "command": "node bin/compile.babel",
-      "env": {
-        "DEBUG": "app:*"
-      }
-    },
-    "dev": {
-      "command": "nodemon --exec node bin/server.babel",
-      "env": {
-        "NODE_ENV": "development",
-        "DEBUG": "app:*"
-      }
-    },
     "build": {
-      "command": "yarn run clean && yarn run compile",
+      "command": "webpack --config webpack/webpack.config --colors --progress --env.env=production",
```

Comment:
i like using betterScripts because it makes it cleaner and more readable to set various env variables specific to a given script...instead of having it all crammed into one line. you can extend anything...so if you did `yarn run build --someCliOption`...it would just append it to what is already defined in the command.

So for example, say you want to only test one file, you could run `yarn run test:specs FileNameToPatternMach`